app/build.gradle

```
@@ -29,9 +32,30 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    dataBinding{
+        enabled = true
+    }
```

```suggestion
    buildFeatures {
        dataBinding true
    }
```
해당 표기법은 옛날 방식 표기법이라서 이렇게 표기하는 게 좋습니당 :) 그리고 `groovy`언어의 경우 =으로 할당하는 게 아니라 한 칸 띄어서 저렇게 값을 할당해주시면 됩니당 :)

---

수정했습니다!!